### PR TITLE
Update capstone_pitch_template.md -- Alex

### PR DESCRIPTION
**Pull request by alex-w-k**

### CODE REVIEW

File: 1703-b/4module/capstone_pitch_template.md [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
 
 Which new techniques are you hoping to implement?
 
+Charts.js, actioncable, possibly HTTP/2
+
 ### Workflow
 
 What project management tool will you be using to manage your progress?
+
+Waffle.io
```
Comment (tmikeschu) [inline 1703-b/4module/capstone_pitch_template.md]: 👍

File: 1703-b/4module/capstone_pitch_template.md [hunk 2]
```diff
@@ -1,27 +1,41 @@
...
 1-3 sentences describing what type of user your app would be applicable to.
 
+My target user is people who run meetups, that want to host a speed dating event but have no idea where to start. It can be a great fundraiser for the group.
+
+
 ### New Techniques
 
 Which new techniques are you hoping to implement?
 
+Charts.js, actioncable, possibly HTTP/2
```
Comment (tmikeschu) [inline 1703-b/4module/capstone_pitch_template.md]: You'll need 1-2 more depending on the HTTP/2 decision. What have you considered?

File: 1703-b/4module/capstone_pitch_template.md [hunk 3]
```diff
@@ -1,27 +1,41 @@
 ### Project Template
 
-### [Project Title]
+### [Inclusive Speed Dating]
 
 ### Pitch
 
 1 sentence that explains the value proposition of the application. How would you explain it to a potential business partner, team member, or investor?
 
+There are only a few speed dating applications out there, but most of them don't account for people that date more than one genders, I am planning on solving that problem as well as making it easy for facilitators to buy access and just click go and host an event.
```
Comment (tmikeschu) [inline 1703-b/4module/capstone_pitch_template.md]: This sounds like a big scope! Be mindful of setting your MVP.

File: 1703-b/4module/capstone_pitch_template.md [hunk 4]
```diff
@@ -1,27 +1,41 @@
...
 1-3 sentences describing the problem that you are trying to solve.
 
+Making it easy for facilitators to run inclusive speed dating events.
+Making sure it is inclusive to all genders/sexualities/abilities.
+
 ### Solution
 
 1-3 sentences describing how your application will solve that problem.
 
+The application will allow people to sign up and specify what genders they are interested in and what level of ability they are. If they are unable to move it will place them in a place where they won't have to move as much and the algorythm will still match them will all available matches.
```
Comment (tmikeschu) [inline 1703-b/4module/capstone_pitch_template.md]: What are the "available matches"? Places or people?

File: 1703-b/4module/capstone_pitch_template.md [hunk 5]
```diff
@@ -1,27 +1,41 @@
...
 
 1-3 sentences describing how your application will solve that problem.
 
+The application will allow people to sign up and specify what genders they are interested in and what level of ability they are. If they are unable to move it will place them in a place where they won't have to move as much and the algorythm will still match them will all available matches. 
+
 ### Target Audience
 
 1-3 sentences describing what type of user your app would be applicable to.
 
+My target user is people who run meetups, that want to host a speed dating event but have no idea where to start. It can be a great fundraiser for the group.
```
Comment (tmikeschu) [inline 1703-b/4module/capstone_pitch_template.md]: 👍

File: 1703-b/4module/capstone_pitch_template.md (hunk 2 above)
Comment (tmikeschu) [reply 1703-b/4module/capstone_pitch_template.md]: How do the charts come into play?

File: 1703-b/4module/capstone_pitch_template.md (hunk 4 above)
Comment (alex-w-k) [reply 1703-b/4module/capstone_pitch_template.md]: That would be people.

File: 1703-b/4module/capstone_pitch_template.md (hunk 2 above)
Comment (alex-w-k) [reply 1703-b/4module/capstone_pitch_template.md]: Charts would be for stats for the person running the events, so they can see how many successful matches, and if any messages and how many were exchanged through the service.

Comment (alex-w-k) [reply 1703-b/4module/capstone_pitch_template.md]: I've also considered using React or Vue.js for the front end if HTTP/2 doesn't end up going well.

Comment (alex-w-k) [reply 1703-b/4module/capstone_pitch_template.md]: Probably using Chef/Inspec/Test-Kitchen/Chef-Spec/Jenkins CI for deployments.

Comment (alex-w-k) [reply 1703-b/4module/capstone_pitch_template.md]: Maybe also using a docker for deployments if it seems like it's useful.

File: 1703-b/4module/capstone_pitch_template.md (hunk 4 above)
Comment (tmikeschu) [reply 1703-b/4module/capstone_pitch_template.md]: The placement algorithm sounds vague to me. Make sure you clearly plan out what that will do.

File: 1703-b/4module/capstone_pitch_template.md (hunk 2 above)
Comment (tmikeschu) [reply 1703-b/4module/capstone_pitch_template.md]: Let's cut the "probably" and "maybe" 's and make some solid plans 👍

**Comment by tmikeschu** [comment]
@alex-w-k start working on your schema ASAP if you haven't already and let me know when you have it.

**Pull request merged**